app/js/components/ConceptDisplay.jsx

```diff
@@ -12,27 +12,19 @@ const {
 } = formValidations;
 
 class ConceptDisplay extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      concept: {},
-    };
-  }
```

Comment:
This code, and the contents of `UNSAFE_componentWillReceiveProps` below, literally did nothing. Only there to be confusing.